### DISCUSSION TITLE
TimeLapse QR set up never starts

### DISCUSSION THREAD

**Discussion by carlosbaca1988** [gopro/labs#186] [General]
I am trying to set my GoPro 11 to do a time lapse during day light ( 8 am to 6pm ) in a period of time of 30 days. 

To do that I am trying first at my place, setting it up for one hour ( some times more sometimes less ). 

When ever I the set up and scan with the GoPro the QR code, it is read properly, and says how much time until Timelapse starts. But it never does it. I just simple doesn’t start. 

The GoPro si connected to a USB so is fully charged at all times but still doesn’t do anything. 

I hace already checked time and date, try different modes ( like video mode ), etc it simply doesn’t turn on when time comes.

thanks in advance

**Comment by dnewman-gpsw** [comment]
The log file might explain what is happing.  Located in the MISC folder on the SD card, called qrlog.txt.  Share here.

**Reply by carlosbaca1988** [reply]
<img width="1512" alt="Screenshot 2023-01-10 at 17 42 27" src="https://user-images.githubusercontent.com/122382009/211611572-6475d030-2be3-4e32-9fc4-d0a524539184.png">
That's on the log not sure if you can get something out of it.

**Comment by carlosbaca1988** [comment]
this is the set up I did on the app for time lapse 
https://www.dropbox.com/s/vcqq2ks8k1rtdgb/RPReplay_Final1673369097.MP4?dl=0

**Comment by dnewman-gpsw** [comment]
You are being hit by a new bug discovered a few days ago.

The workaround.  Simplify your command, and do the preset setup separately.

mTPdTp.10fLw55cFi4M1  (the rest of you commands don't apply to timelapse photos.)  Make sure you have a Timelapse Photo preset, the camera defaults to video TL.

Then your daily timelapse will be !08:00SQ!36000E!R

The random shutdown you are seeing will be fixed in the next Labs firmware.  The bug only affects delay captures like this, when you are take too much time before starting a record.

**Comment by carlosbaca1988** [comment]
Sorry but I don't know how to simplify the command and do the preset setup separately. could you specify a bit. 

also I need this part : mTPdTp.10fLw55cFi4M1 (the rest of you commands don't apply to timelapse photos.) Make sure you have a Timelapse Photo preset, the camera defaults to video TL.

to be explain in a easier way. sorry in advance. 

thanks!

**Comment by dnewman-gpsw** [comment]
All commands are just text, put the text into a QR code, via the "Added Command" or click these links 

Setup
https://gopro.github.io/labs/control/set/?cmd=mTPdTp.10fLw55cFi4M1

and 

Timelapse
https://gopro.github.io/labs/control/set/?cmd=!08:00SQ!36000E!R

**Comment by carlosbaca1988** [comment]
I have been trying all modes and what I can see is that the error happens when I set a time to start ( which is what I need )

I need to be filming from 8 am to 8 pm every day during a month period.

So, in your link when I scan it it says it will start at 8 am, but is it set to stop in the evening and repeat action every day?

Will test tomorrow anyways.

thanks for the help

**Comment by carlosbaca1988** [comment]
I understand now about using the preset and not adding more code. I did that , and the problem seems to still the same. I can't set a time to start with GoPro labs ( if its with the GoPro yes ) but as I said , I need it to start a stop at a certain time as I don't need it to bee shooting during the night.

**Reply by dnewman-gpsw** [reply]
Looks like you will have to wait for the next firmware release, as it restores this functionality.
